src/BlockchainWriter/Bitcoin.test.ts

```diff
@@ -46,21 +49,26 @@ describe('Bitcoin.getData', async assert => {
       prefix: PREFIX_POET,
       version: [0, 2],
       storageProtocol: StorageProtocol.IPFS,
-      ipfsDirectoryHash: 'ipfsDirectoryHash',
+      ipfsDirectoryHash: getB58('ipfsDirectoryHash'),
     }
 
     testGetData(poetAnchor)
-    testGetData({ ...poetAnchor, prefix: PREFIX_BARD, ipfsDirectoryHash: 'another ipfsDirectoryHash' })
+    testGetData({ ...poetAnchor, prefix: PREFIX_BARD, ipfsDirectoryHash: getB58('another ipfsDirectoryHash') })
```

Comment:
I don't think you really need to run `getB58` on this input...

If it's failing because it tries to parse it as a BS58 string, can you just change it to an actual IPFS hash instead?

Reply:
Yes, it fails because tries to read BS58 string.

`UnhandledPromiseRejectionWarning: Error: Non-base58 character`

ok, I will change for a real IPFS hash.